deploy/playbooks/roles/celery/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: add rabbitmq official repo to sources.list
+  sudo: yes
+  lineinfile:
+     dest: /etc/apt/sources.list 
+     regexp: '^deb http://www.rabbitmq.com/debian/ testing main'
+     line: 'deb http://www.rabbitmq.com/debian/ testing main'
+     backrefs: yes
```

Comment:
can we use `/etc/sources.list.d` here, instead? IMHO t's easier to spot the repo that way